ArjunaJTS/jtax/tests/classes/com/hp/mwtests/ts/jta/jts/tools/JTSOSBTestBase.java

```diff
@@ -132,20 +144,27 @@ protected void assertBeanWasCreated(ArjunaTransactionImple txn) {
 
         MBeanServer mbs = JMXServer.getAgent().getServer();
 
+        showAllMBeans(mbs);
+
 		try {
-			String type = txn.type();
+			String type = txn.type().replace(File.separator, "/");
+
 			if (type.charAt(0) == '/')
```

Comment:
Ah got you, was too lazy to check the syntax of replace... I thought you're changing "/" to File.separator.